### DISCUSSION TITLE
Picture freezing with Telenor IPTV | Philips Hue

### DISCUSSION THREAD

**Discussion by nogu91** [fabianishere/udm-iptv#94] [Q&A]
Hi,

I have successfully installed the service, and after adding a rule to accept traffic UDP ports 5500 and 8000 to the following destination addresses: 233.184.48.0/24, 233.184.49.0/24, 233.171.129.0/24, IPTV started working. However, the picture freezes after a couple of minutes. I have tried opening for all IGMP traffic in the firewall with no luck. What am I doing wrong?

=== Configuration ===
WAN Interface: eth4
WAN Ranges: 224.0.0.0/4 93.91.111.0/24 148.122.7.125
WAN VLAN: 0 (dev iptv)
WAN DHCP options: k
LAN Interfaces: br0
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: true
=== IP Link and Route ===
4: eth4: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc htb qlen 1000
    inet 80.212.134.19/22 scope global dynamic eth4
       valid_lft 1127sec preferred_lft 1127sec
80.212.132.0/22 dev eth4  src 80.212.134.19
=== Service Logs ===
Mar 10 15:52:52 ubnt udm-iptv[18910]: The IGMP message was local multicast. Ignoring.
Mar 10 15:52:52 ubnt udm-iptv[18910]: RECV V2 member report   from 192.168.1.190   to 239.255.3.22
Mar 10 15:52:52 ubnt udm-iptv[18910]: Updated route entry for 239.255.3.22 on VIF #1
Mar 10 15:52:53 ubnt udm-iptv[18910]: The IGMP message was local multicast. Ignoring.
Mar 10 15:52:53 ubnt udm-iptv[18910]: RECV V2 member report   from 192.168.1.91    to 233.184.48.94
Mar 10 15:52:53 ubnt udm-iptv[18910]: Updated route entry for 233.184.48.94 on VIF #1
Mar 10 15:52:53 ubnt udm-iptv[18910]: Adding MFC: 93.91.111.2 -> 233.184.48.94, InpVIf: 0
Mar 10 15:52:53 ubnt udm-iptv[18910]: RECV V2 member report   from 192.168.1.91    to 233.184.48.56
Mar 10 15:52:53 ubnt udm-iptv[18910]: Updated route entry for 233.184.48.56 on VIF #1
Mar 10 15:52:53 ubnt udm-iptv[18910]: Adding MFC: 93.91.111.2 -> 233.184.48.56, InpVIf: 0

**Comment by nogu91** [comment]
Update on freezing picture:
Now it seems like the freezing picture problem has been solved after changing the IGMP rule type from "Internet In" to "Internet Local", but a new problem has emerged.

New problem:
I have a Philips Hue system installed in my apartment and it seems like the bridge somehow manages to join in on the same multicast group as the decoder, as there is a constant traffic that is forwarded to both of the LAN ports as long as the decoder is connected to power. If I disconnect power to the decoder, all traffic to the Hue bridge stops as well. I assume that this could be solved by setting up a separate VLAN for IPTV, but I would like to avoid this as some services from Telenor (e.g. viewing content through the mobile app) are dependent on the decoder being discoverable on the LAN. Is there any other ways I could approach this?

**Reply by fabianishere** [reply]
Could you try removing `224.0.0.0/4` from `IPTV_WAN_RANGES` in `/etc/udm-iptv.conf`?

**Reply by nogu91** [reply]
That didn't work unfortunately, but it seems like the range is not needed in the config either, as everything still works as before after it was removed in both config and FW. I have also mDNS and IGMP Snooping enabled on the network

**Reply by nogu91** [reply]
Now I have tried to connect my TV to the UDM as well, using a cable, and the same problem appears here as well. However, if I connect the same TV using WiFi, there is no traffic. I'm thinking that this could mean that there is something with the switch that isn't working as it should, but I am not that proficient in networking, meaning that if anyone has any suggestions for how to proceed with troubleshooting this, I would be really grateful!

**Reply by fabianishere** [reply]
I recommend doing a packet trace of your LAN to see what's causing the traffic:

```bash
tcpdump -i br0 -w trace.pcap
```
